iModelCore/BeSQLite/PublicAPI/BeSQLite/BeSQLite.h

```diff
@@ -2523,6 +2523,9 @@ struct EXPORT_VTABLE_ATTRIBUTE Db : NonCopyableClass
         // Skip the check for SQLite file validity before opening. When using the CloudSqlite mode, the local file is not in SQLite normal format.
         bool m_skipFileCheck = false;
 
+        // Provide uri to shared schema channel use by default during schema upgrade or update.
+        Utf8String m_sharedSchemaChannelUri;

```

Comment:
what is this uri used for?

Reply:
It's path for cloud sqlite file and parameters like key require to access it.

Reply:
that won't work. We need to open this database from JavaScript and pass it in where it's needed.

Reply:
I need to ATTACH the SQLite db to the primary connection. This is the fastest way to sync schemas.